src/lib/cron-tab/custom.tsx

```diff
@@ -4,18 +4,20 @@ interface CustomCronProp {
     onChange(e: string[]): void
     value: string[]
     translate(e: string): string
+    isDisabled?: boolean
```

Comment:
can you change it to generic key "disabled" ?

Reply:
Okay